components/fieldgroup/stories/template.js

```diff
@@ -20,7 +21,23 @@ export const Template = (
 		labelPosition,
 		isInvalid,
 		helpText,
-		items = [],
+		items = [
+			{
+				id: "apple",
+				label: "Apples are best",
+				customClasses: ["spectrum-FieldGroup-item"],
+			},
+			{
+				id: "banana",
+				label: "Bananas forever",
+				customClasses: ["spectrum-FieldGroup-item"],
+			},
+			{
+				id: "pear",
+				label: "Pears or bust",
+				customClasses: ["spectrum-FieldGroup-item"],
+			}
+		],
```

Comment:
If you pass _this_ as the default `items` args, does that work? So in the story file, we'd have to remove all of the `(passthroughs, context) => Radio({...passthroughs,` etc? Trying it really quickly, I think that should work still.

And then this can remain `items = []` in the template.